tests/fv3_conf/rrfs_warm_run.IN

```diff
@@ -0,0 +1,30 @@
+rm -fr INPUT RESTART
+mkdir INPUT RESTART
+
+OPNREQ_TEST=${OPNREQ_TEST:-false}
+SUFFIX=${RT_SUFFIX}
+cp -r @[INPUTDATA_ROOT]/FV3_input_data_conus13km/INPUT/* INPUT/
```

Comment:
@SamuelTrahanNOAA where is your FV3_input-data_conus13km located? We need to copy it to current INPUTDATA_ROOT directory on UFS supported platforms. Are there any other input files?

Reply:
/scratch2/BMC/wrfruc/Samuel.Trahan/RT/NEMSfv3gfs/input-data-20211210/FV3_input_data_conus13km

Reply:
Also /scratch2/BMC/wrfruc/Samuel.Trahan/RT/NEMSfv3gfs/input-data-20211210/FV3_fix/seaice_newland.grb